nncore/models/loss/celoss.py

```diff
@@ -0,0 +1,23 @@
+from typing import Dict
+from torch.functional import Tensor
+from torch.nn import functional as F
+
+from torch import nn
+import torch
+
+
+class CEwithstat(nn.Module):
+    r"""CEwithstat is warper of cross-entropy loss"""
```

Comment:
Is it typo error here? `warper` -> `wrapper`
Please use camel case for class names